IFC4x3/Properties/d/Depth_19jxXCHUr5VhPM7hPVeFYR/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Depth_19jxXCHUr5VhPM7hPVeFYR" Name="Depth" UniqueId="49b7b84c-45ed-457e-b656-1eb65fa0f89b" PrimaryDataType="IfcPositiveLengthMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Depth" UniqueId="49b7b84c-45ed-457e-b656-1eb65fa0f89b" PrimaryDataType="IfcPositiveLengthMeasure" />
 
```

Comment:
Is Depth orphaned?

Reply:
Yes, any property without a href id is orphaned.